components/inputs/input-date.js

```diff
@@ -237,13 +240,20 @@ class InputDate extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixin(
 		const formattedWideDate = formatISODateInUserCalDescriptor('2323-12-23');
 		const shortDateFormat = (this._dateTimeDescriptor.formats.dateFormats.short).toUpperCase();
 
-		const clearButton = !this.required ? html`<d2l-button-subtle text="${this.localize(`${this._namespace}.clear`)}" @click="${this._handleClear}"></d2l-button-subtle>` : null;
-		const nowButton = this.hasNow ? html`<d2l-button-subtle text="${this.localize(`${this._namespace}.now`)}" @click="${this._handleSetToNow}"></d2l-button-subtle>` : null;
+		const clearButton = !this.required ? html`<d2l-button-subtle text="${this.localize('components.input-date.clear')}" @click="${this._handleClear}"></d2l-button-subtle>` : null;
+		const nowButton = this.hasNow ? html`<d2l-button-subtle text="${this.localize('components.input-date.now')}" @click="${this._handleSetToNow}"></d2l-button-subtle>` : null;
 		const icon = (this.invalid || this.childErrors.size > 0)
 			? html`<d2l-icon icon="tier1:alert" slot="left" style="${styleMap({ color: 'var(--d2l-color-cinnabar)' })}"></d2l-icon>`
 			: html`<d2l-icon icon="tier1:calendar" slot="left"></d2l-icon>`;
 		const errorTooltip = (this.validationError && !this.opened && this.childErrors.size === 0) ? html`<d2l-tooltip align="start" announced for="${this._inputId}" state="error" class="vdiff-target">${this.validationError}</d2l-tooltip>` : null;
 
+		const instructions = this.localize('components.input-date.openInstructions', { format: shortDateFormat });
+		const revertMessageInstructionsTooltip = this.localize('components.input-date.revertWithInstructions', { label: this.label, format: shortDateFormat });
+		const instructionsMessage = this._showRevertInstructions ? revertMessageInstructionsTooltip : instructions;
+
+		const revertMessageRevertTooltip = this.localize('components.input-date.revertWithFormat', { label: this.label, format: shortDateFormat });
+		const revertTooltip = (this._showRevertTooltip && !this.opened) ? html`<d2l-tooltip align="start" force-show announced class="vdiff-target" for="${this._inputId}">${revertMessageRevertTooltip}</d2l-tooltip>` : null;
```

Comment:
The logic here:
- `_showRevertTooltip` gets set to `true` if there's an error thrown in `_handleChange`
- When  `_showRevertTooltip` is true, we show the tooltip with the "[field] was reverted. Use X format" message. This appears when the user blurs from the `input-text`
- If the user then re-focuses on the `input-text`, the instructions message has "[field] was reverted" appended to the start.
- This only appears on first re-focus and not subsequent ones unless the user inputs another incorrectly formatted date

This seems to work but let me know if anyone has ideas for a simpler algorithm.